contents/tutorials/property-filter.md

```diff
@@ -44,6 +44,30 @@ To configure the app to remove selected properties, simply select the blue gear
 
 ![PostHog Property Filter](../images/tutorials/property-filter/property-filter-tutorial-2.png)
 
+## The full list of GeoIP properties
+
+...at time of writing are
+
+* $geoip_city_name
+* $geoip_country_name
+* $geoip_country_code
+* $geoip_continent_name
+* $geoip_continent_code
+* $geoip_postal_code
+* $geoip_latitude
+* $geoip_longitude
+* $geoip_time_zone
+* $geoip_subdivision_1_name
+* $geoip_subdivision_1_code
+* $geoip_subdivision_2_name
+* $geoip_subdivision_2_code
+* $geoip_subdivision_3_code
+* $geoip_subdivision_3_name
+
+You can [check the current list in the source code for the app.](https://github.com/PostHog/posthog-plugin-geoip)
+
+## A working example
```

Comment:
.. since you're at it, can we also add removing a `$set` person property?

So we can avoid this in the future: https://github.com/PostHog/posthog-plugin-geoip/pull/19

Basically, `$set.$geoip_city_name` removes the city name from `$set` properties.

Reply:
I _think_ I've got that 🙈

Reply:
lovely, thanks!